### PR TITLE
Update jobapi pt example

### PR DESCRIPTION
**Pull request by ZiyueXu77**
Fixes # .

### Description

correct cifar10 download issue, update learner to clientapi

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: examples/advanced/job_api/pt/src/cifar10_fl_partitioned.py [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+                # (optional) use GPU to speed things up
+                images, labels = data[0].to(DEVICE), data[1].to(DEVICE)
+                # calculate outputs by running images through the network
+                outputs = net(images)
+                # the class with the highest energy is what we choose as prediction
+                _, predicted = torch.max(outputs.data, 1)
+                total += labels.size(0)
+                correct += (predicted == labels).sum().item()
+
+        return 100 * correct // total
```
[the remark this answers is not in the document]
Comment (ZiyueXu77) [reply examples/advanced/job_api/pt/src/cifar10_fl_partitioned.py]: makes sense updated

File: examples/advanced/job_api/pt/fedavg_script_runner_xsite_val_cifar10.py [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    # Set up model locator for cross-site evaluation
+    model_locator_id = job.to_server(PTFileModelLocator(pt_persistor_id=persistor_id), id="model_locator")
+
+    # Define the cross-site model evaluation workflow
+    xsite_eval_controller = CrossSiteModelEval(
+        model_locator_id=model_locator_id,
+        submit_model_timeout=600,
+        validation_timeout=6000,
+        cleanup_models=False,
+        validation_task_name="validate",
```
[the remark this answers is not in the document]
Comment (ZiyueXu77) [reply examples/advanced/job_api/pt/fedavg_script_runner_xsite_val_cifar10.py]: good catch, updated

File: examples/advanced/job_api/pt/fedavg_script_runner_xsite_val_cifar10.py [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+def load_cifar10_labels():
+    """Load CIFAR10 training labels for data splitting."""
+    import torchvision
+    import torchvision.transforms as transforms
+
+    dataset_path = "/tmp/nvflare/data/cifar10"
+    transform = transforms.Compose([transforms.ToTensor()])
+    trainset = torchvision.datasets.CIFAR10(root=dataset_path, train=True, download=True, transform=transform)
+    labels = np.array([label for _, label in trainset])
+    return labels
```
[the remark this answers is not in the document]
Comment (ZiyueXu77) [reply examples/advanced/job_api/pt/fedavg_script_runner_xsite_val_cifar10.py:51]: added

**Comment by ZiyueXu77** [comment]
> Additional Comments (1)
> **`examples/advanced/job_api/pt/src/cifar10_fl_train_eval_submit.py`** Integer division loses precision in accuracy calculation
> 
> ```
>         return 100.0 * correct / total
> ```

In Python 3, / already does floating‑point division. To force float calculation explicitly,  the float literal 100.0 * correct / total is the simplest and clearest

**Comment by ZiyueXu77** [comment]
> Additional Comments (3)
> **`examples/advanced/job_api/pt/src/cifar10_fl.py`** Integer division loses precision in accuracy calculation
> 
> ```
>             return 100.0 * correct / total
> ```
> 
> **`examples/advanced/job_api/pt/src/cifar10_fl.py`** Integer division loses precision in displayed accuracy
> 
> ```
>             print(f"Accuracy of the network on the 10000 test images: {100.0 * correct / total} %")
> ```
> 
> **`examples/advanced/job_api/pt/src/cifar10_fl_train_eval_submit.py`** Integer division loses precision in accuracy calculation
> 
> ```
>         return 100.0 * correct / total
> ```

@greptileai, it will not loss precision, test yourself:

python -c "correct=3; total=8; print(100*correct/total, 100.0*correct/total, 100*float(correct)/total)"

File: examples/advanced/job_api/pt/swarm_script_runner_cifar10.py [hunk 4]
```diff
@@ -19,14 +19,19 @@
...
 
 if __name__ == "__main__":
     n_clients = 2
     num_rounds = 3
-    train_script = "src/train_eval_submit.py"
+    train_script = "src/cifar10_fl_partitioned.py"
 
     job = CCWFJob(name="cifar10_swarm")
+
+    # Add TensorBoard receiver to all clients
```
[the remark this answers is not in the document]
Comment (ZiyueXu77) [reply examples/advanced/job_api/pt/swarm_script_runner_cifar10.py:32]: no it is imported on line 22

File: examples/advanced/job_api/pt/cyclic_cc_script_runner_cifar10.py [hunk 5]
```diff
@@ -26,6 +27,9 @@
 
     job = CCWFJob(name="cifar10_cyclic")
 
+    # Add TensorBoard receiver to all clients
```
[the remark this answers is not in the document]
Comment (ZiyueXu77) [reply examples/advanced/job_api/pt/cyclic_cc_script_runner_cifar10.py:30]: no it is imported on line 20

**Review by YuanTingHsieh — APPROVED** [review]
